packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -110,39 +121,47 @@ export class WaferMap extends FoundationElement {
      */
     public render(): void {
```

Comment:
This is only used inside this file and can be made private

Reply:
It is being used also in the tests (`wafer-map.spec.ts`) by the jasmine `spy` to track the rendering process. For now, I think it would be best to keep it `public `but marked as `internal`.